.github/travis.php

```diff
@@ -50,4 +50,4 @@
     }
 }
 
-file_put_contents('packages.json', json_encode(compact('packages'), $flags));
+file_put_contents(dirname(__DIR__).'/packages.json', json_encode(compact('packages'), $flags));
```

Comment:
yep, I'm debugging travis right now, this PR triggers situations that are not handled correctly today, please don't merge until it's green. I'll post a notice here.